pkg/assembler/backends/ent/schema/certifyvex.go

```diff
@@ -0,0 +1,47 @@
+package schema
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/dialect/entsql"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/index"
+)
+
+// CertifyVex holds the schema definition for the CertifyVex entity.
+type CertifyVex struct {
+	ent.Schema
+}
+
+// Fields of the VEX.
+func (CertifyVex) Fields() []ent.Field {
+	return []ent.Field{
+		field.Int("package_id").Optional().Nillable(),
+		field.Int("artifact_id").Optional().Nillable(),
+		field.Int("vulnerability_id").Comment("Vulnerability is one of OSV, GHSA, or CVE, or nil if not vulnerable"),
+		field.Time("knownSince"),
```

Comment:
Ent keys need to be snake_case in order to generate correct columns in postgres which is case sensitive.

Reply:
Thanks, I didn't know.
Fixed it in https://github.com/ivanvanderbyl/guac/pull/5/commits/ecd262aaa794683b48e7617798ef29471d6ace58